README.md

```diff
@@ -11,60 +11,70 @@ on IBM Blockchain.  The components of the demo are:
-4. Make sure the key/value store only has values for your current network (See below).
-5. Run these commands in the cloned directory:
+1. Deploy the app on my local machine, connecting to an IBM Blockchain network running in Bluemix  - [instructions](#manbluenetwork)
 
-```shell
-npm install
-gulp
-```
 
-These credentials can be obtained from the "Service Credentials" tab of the Bluemix service. They are
-in the form:
-
-```json
-{
-  "credentials": {
-    "peers": [
-      {
-        "discovery_host": "169.53.62.121",
-        "discovery_port": "40275",
-        "api_host": "169.53.62.121",
-        "api_port": "40276",
-        "type": "peer",
-        "network_id": "4b21f2f9-4d10-4946-a0df-f91ac09dbc03",
-        "id": "4b21f2f9-4d10-4946-a0df-f91ac09dbc03_vp1",
-        "api_url": "http://169.53.62.121:40276"
-      }
-    ],
-    "users": [
-      {
-        "username": "user_type0_b7c7a1e545",
-        "secret": "89ce33e4e6"
-      }
-    ]
-  }
-}
-```
+# <a name="manbluenetwork"></a>Deploy the CP-Web App locally and connect to an IBM Blockchain Network running in Bluemix:
+
+1.  Follow these instructions to [Set up your environment for running the demos](https://github.com/ptippett/marbles/blob/break_out_common_sections/demo_prereqs)
+
+1.  Clone the CP-Web app to your local system so you can run it here
+To do this, run ```git clone http://gopkg.in/ibm-blockchain/cp-web.v2``` to clone the v2.0 branch to your local system.  
+
+1.  Follow the instructions to [Set up a new bluemix network or grab credentials from an existing network](https://github.com/ptippett/marbles/blob/break_out_common_sections/create_blockchain_bluemix.md)
 
-## Using the Demo
-1. Register some users using the registration form on the login page.
+
+1. Make sure the key/value store only has values for your current network (See below).
+1. Run these commands in the cloned directory (typically ```<git location>/cp-web```).
+  1.  If you're running on windows, you need to install some additional dependencies.
+    1.  Run ```npm install --global windows-build-tools``` to install the dependencies listed [here](https://github.com/felixrieseberg/windows-build-tools)
+    1.  Verify you have a `c:\tmp` directory and create it if not.  `hfc` uses this folder to temporarily store and package this demo's chaincode for deployment.
+  2. On linux, or after you've installed build tools on windows, then run  
+     `npm install`  
+     `gulp`  
+     
+1. If all goes well you should see this message in the console:
+	
+		--------------------------------- Server Up - localhost:3000 ------------------------------------
+		
+1. The app is already coded to auto deploy the chaincode.  You should see further message about it deploying.
+ **[IMPORTANT]** You will need to wait about 60 seconds for the cc to fully deploy. The SDK will do the waiting for us by stalling our callback.
+ 
+1. Once you see this message you are good to go: 
+		
+		[ibc-js] Deploying Chaincode - Complete
```

Comment:
Good catch.  I just stole that form the previous documentation I think, but yeah the message you pasted is what I saw when I ran through this. I'll update.